pkg/runtime/inject/inject.go

```diff
@@ -100,21 +100,6 @@ func SchemeInto(scheme *runtime.Scheme, i interface{}) (bool, error) {
 	return false, nil
 }
 
-// Stoppable is used by the ControllerManager to inject stop channel into Sources,
-// EventHandlers, Predicates, and Reconciles.
-type Stoppable interface {
-	InjectStopChannel(<-chan struct{}) error
```

Comment:
As mentioned elsewhere, I don't think we should just remove it, because that will result in hard to debug panics down the line.

Reply:
I'd rather have the context properly propagated within reconciler calls, rather than having depedency injection like these around, which would result in the context to be stored as a struct field

Reply:
(See reply above, we can inject ctx.Done())